docs/compatibility.md

```diff
@@ -1,5 +1,7 @@
 # Compatibility
 
+## Executable "apps"
+
 The following apps are supported by govuk-docker to some extent.
```

Comment:
This line should change (or be removed) if we're also including things that aren't supported.

Reply:
I've rewritten this based on the other comments.